### PR TITLE
IGAPP-1028: Poi markers pixelated

### PR DESCRIPTION
**Pull request by f1sh1918**
1. http://localhost:9000/testumgebung/de/locations
2. Zoom on the map and check if markers are still pixelated

Tested on web & ios

### CODE REVIEW

**Review by Cmd190 — APPROVED** [review]
Looks fine.

File: release-notes/unreleased/IGAPP-1028-poi-markers-pixelated.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+issue_key: IGAPP-1028
+show_in_stores: true
+platforms:
+  - web
```
Comment (steffenkleinle) [inline release-notes/unreleased/IGAPP-1028-poi-markers-pixelated.yml:4]: Is this really web only? I think the used icons are changed on native as well, right?

File: api-client/src/maps/index.ts [hunk 2]
```diff
@@ -84,7 +84,7 @@ export const animationDuration = 2000
 export const nameQueryParam = 'name'
 
 export const mapMarker: MarkerConfig = {
-  iconSize: 1.75,
-  symbol: 'marker_15',
-  symbolActive: 'marker_15_active'
+  iconSize: 0.75,
```
Comment (steffenkleinle) [inline api-client/src/maps/index.ts:87]: Not sure and maybe check back with the design team but maybe the icons are a little bit small now?

**Review by steffenkleinle — APPROVED** [review]
Tested on firefox and android

File: api-client/src/maps/index.ts (hunk 2 above)
Comment (f1sh1918) [reply api-client/src/maps/index.ts:87]: well the icons size should be increased by 10%. Are you talking about web or native?

Comment (steffenkleinle) [reply api-client/src/maps/index.ts:87]: Ok. isn't even that smaller than before. I was just confused by `iconSize: 1.75` and `iconSize: 0.75` and thought it looks quite small now. Your choice if something has to be done here.

Comment (f1sh1918) [reply api-client/src/maps/index.ts:87]: no i increased it a little bit to have slightly bigger markers than the other icons on the map. We increased the svgs from 15 to 40 and these svs were converted to spriteicons. so if you compare `15*1.75 and 40*0.75` you can get the size by pixels. I think if thats not fine we gonna get the design feedback during beta

File: release-notes/unreleased/IGAPP-1028-poi-markers-pixelated.yml (hunk 1 above)
Comment (f1sh1918) [reply release-notes/unreleased/IGAPP-1028-poi-markers-pixelated.yml:4]: gonna adjust it

**Pull request merged**